first_timers/first_timers.py

```diff
@@ -23,7 +24,7 @@ def humanize_url(api_url):
     return human_url_template.format(user=user, repo=repo, issue_num=issue_num)
 
 
-def get_first_timer_issues():
+def get_first_timer_issues(daysold=15):
```

Comment:
also, lets move `15` to top of the file at line 9 or somewhere around that as a constant value `DAYS_OLD=15` so it will be easy to configure and the function would look like

```
def get_first_timer_issues(days_old=DAYS_OLD):
```